app/assets/stylesheets/ltags/UserSubscriptionTag.scss

```diff
@@ -1,85 +1,74 @@
+@import 'config/import';
+
 .ltag__user-subscription-tag {
-  .ltag__user-subscription-tag__container {
+  &__container {
+    border-radius: var(--radius);
+    border: 2px solid var(--body-color);
+    box-shadow: 4px 4px 0 var(--body-color);
+    box-sizing: border-box;
     display: flex;
     justify-content: center;
-    border: 2px solid black;
-    box-shadow: 4px 4px 0 black;
-  }
-
-  #profile-images {
-    text-align: center;
+    padding: var(--su-4);
 
-    img {
-      border-radius: 50%;
+    @media screen and (min-width: $breakpoint-m) {
+      padding: var(--su-6);
     }
   }
 
-  .ltag__user-subscription-tag__subscriber-profile-image {
-    display: none;
-  }
-
-  .ltag__user-subscription-tag__content {
-    margin: 1rem auto;
+  &__signed-in {
+    color: var(--base-70);
   }
 
-  .ltag__user-subscription-tag__cta-text {
-    text-align: center;
-  }
+  &__profile-images {
 
-  .ltag__user-subscription-tag__signed-out {
-    text-align: center;
-  }
+    &.signed-in {
+      margin: 0 auto var(--su-4);
+      position: relative;
+      height: 80px;
+      width: 80px;
+    }
 
-  .ltag__user-subscription-tag__response {
-    display: none;
-  }
+    &.signed-out {
+      height: auto;
+      width: auto;
+      text-align: center;
+      margin: 0 auto;
 
-  #sign-in-btn {
-    width: max-content;
-  }
+      .ltag__user-subscription-tag__author-profile-image {
+        margin: 0 auto var(--su-4);
+      }
+    }
 
-  .ltag__user-subscription-tag__signed-in {
-    text-align: center;
-    display: none;
+    img {
+      border-radius: 100%;
+    }
   }
 
-  #logged-in-text,
-  #apple-auth-message {
-    text-align: center;
+  &__author-profile-image,
+  &__subscriber-profile-image {
+    display: block;
   }
 
-  .ltag__user-subscription-tag__subscriber-email {
-    font-weight: bold;
+  &__subscriber-profile-image {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    box-shadow: 0 0 0 4px var(--body-color-inverted);
   }
+}
 
-  #user-subscription-confirmation-modal {
-    display: none;
+// modal styles overwrite crayons component WIP
+#user-subscription-confirmation-modal {
+  .crayons-modal__box__header {
+    padding: var(--su-3) var(--su-4);
   }
 
-  #subscriber-apple-auth {
-    display: none;
+  .crayons-modal__box__body {
+    padding: var(--su-6);
   }
 
   .crayons-modal__overlay {
     position: fixed;
     z-index: 400;
   }
```

Comment:
This "hack" can be removed with these sweet updates 🎉

Reply:
Removed :)